cylc/flow/scheduler.py

```diff
@@ -1499,7 +1500,7 @@ def release_queued_tasks(self) -> bool:
             pre_prep_tasks,
             self.server.curve_auth,
             self.server.client_pub_key_dir,
-            is_simulation=(self.get_run_mode() == RunMode.SIMULATION)
+            run_mode=self.config.run_mode()
```

Comment:
This partially reverts [`db215d3`](https://github.com/cylc/cylc-flow/commit/db215d330f50251c734e4040ae3a61fca673af29#diff-90711b005fe27b9f894e6cd09f08816d847dcf853518894c6b7e2139784cbcb8L1500-R1488).

For consistency I changed these to use the scheduler's method that I added

```suggestion
            run_mode=self.get_run_mode()
```